src/components/FilterTags/FilterTags.js

```diff
@@ -87,20 +88,95 @@ const FilterTags = () => {
     };
   }, []);
 
-  const handleFilterClick = () => {
-    console.log("Filter clicked");
+  /**
+  Apply a given filter
+  @param {object} e The event object
+  @param {string} filter The name of the filter
+   @param {string} filterBy The name of the filter group
+*/
+  const addActiveFilter = (e, filter, filterBy) => {
+    setActiveFilters(filters => {
+      const updatedFilters = { ...filters };
```

Comment:
Using a `set` might be more appropriate here.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set